pkg/odo/cli/init/init.go

```diff
@@ -125,6 +125,18 @@ func (o *InitOptions) Run() (err error) {
 		}
 	}()
 
+	isEmptyDir, err := location.DirIsEmpty(o.clientset.FS, o.contextDir)
+	if err != nil {
+		return err
+	}
+	if isEmptyDir && len(o.flags) == 0 {
+		fmt.Println("The current directory is empty.")
+		fmt.Println("Odo will help with you starting a new project.")
```

Comment:
`odo` should not be capitalized

Please avoid using newline / merge these messages into one line :)

Reply:
Correct grammar would be` odo will help you start a new project`

Reply:
Ok, I'll change those lines. I copied the messages provided in the parent issue #5494.
Thanks.